### PR TITLE
update the deploy and exec producedure

### PR DESCRIPTION
**Pull request by pursonchen**
update the deploy and exec producedure

### CODE REVIEW

File: Deployment_Procedure.md [hunk 1]
```diff
@@ -49,73 +51,244 @@ Create `eosio.forum`, set contract, and set permission
...
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig approve createaudito '{"actor":"BP_name","permission":"active"}' -p BP_name
-
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig approve createforum '{"actor":"BP_name","permission":"active"}' -p BP_name
-
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig approve createescrow '{"actor":"BP_name","permission":"active"}' -p BP_na
+```shell
+eosc multisig approve pursonchen22 createbetbos <BP ACCOUNT>
```
Comment (DenisCarriere) [inline Deployment_Procedure.md]: either use `bosc` or use the `-u` BOS endpoint flag

Comment (pursonchen) [reply Deployment_Procedure.md]: agree with you

**Comment by DenisCarriere** [comment]
If you can add a section:

### How to create an alias

Add to your `.bashrc` or `.zshrc` file

```bash
alias bosc="eosc -u https://bos.eoshenzhen.io:9443"
```

@pursonchen

File: Deployment_Procedure.md [hunk 2]
```diff
@@ -124,76 +297,13 @@ cleos -u https://bos.eoshenzhen.io:9443 multisig exec pursonpurson createforum -
...
-
-
-cleos -u https://bos.eoshenzhen.io:9443 set contract auditor.bos auditor.bos  -p  auditor.bos@active
+```shell
+eosc system setcontract eosio.forum eosio.forum eosio.forum  
 
+eosc system setcontract auditor.bos auditor.bos auditor.bos
 
-cleos -u https://bos.eoshenzhen.io:9443 set contract escrow.bos escrow.bos  -p  escrow.bos@activ
+eosc system setcontract escrow.bos escrow.bos escrow.bos
```
Comment (DenisCarriere) [inline Deployment_Procedure.md]: Need to include `.wasm` & `.abi` extension

```bash
bosc system setcontract escrow.bos escrow.bos.wasm escrow.bos.abi
```

File: Deployment_Procedure.md [hunk 3]
```diff
@@ -124,76 +297,13 @@ cleos -u https://bos.eoshenzhen.io:9443 multisig exec pursonpurson createforum -
 
 ### `eosio.forum`, `auditor.bos`, `escrow.bos` 
 
-```
-cleos -u https://bos.eoshenzhen.io:9443 set contract eosio.forum eosio.forum -p eosio.forum@active
-
-
-cleos -u https://bos.eoshenzhen.io:9443 set contract auditor.bos auditor.bos  -p  auditor.bos@active
+```shell
+eosc system setcontract eosio.forum eosio.forum eosio.forum
```
Comment (DenisCarriere) [inline Deployment_Procedure.md]: Need to include .wasm & .abi extension

```
bosc system setcontract eosio.forum eosio.forum.wasm eosio.forum.abi
```

**Pull request merged**